tests/test_cmd_base.py

```diff
@@ -121,15 +121,26 @@ def test_failCall(self):
 
 
 class TestModuleTaskLoader(object):
-    def test_load_tasks(self):
+    def test_load_tasks_from_dict(self):
         cmd = Command()
         members = {'task_xxx1': lambda : {'actions':[]},
                    'task_no': 'strings are not tasks',
                    'blabla': lambda :None,
                    'DOIT_CONFIG': {'verbose': 2},
                    }
         loader = ModuleTaskLoader(members)
-        task_list, config = loader.load_tasks(cmd, {}, [])
+        loader.setup({})
+        config = loader.load_doit_config()
+        task_list = loader.load_tasks(cmd, [])
+        assert ['xxx1'] == [t.name for t in task_list]
+        assert {'verbose': 2} == config
+
+    def test_load_tasks_from_module(self):
+        import tests.module_with_tasks as module
```

Comment:
Please put ``import`` on top of file.

Reply:
Are you sure? This is a module needed for this particular test function only. I strongly suggest to keep it local. It is not uncommon at all to have function local imports in test code.

Please let me know.

Reply:
ok, keep it local